content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -16,207 +16,215 @@ aliases:
+    ```bash
+    RESPONSE=$(curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
+                --data-urlencode "grant_type=client_credentials" \
+                --data-urlencode "client_id=${CLIENT_ID}" \
+                --data-urlencode "client_secret=${SECRET}")
+    echo $RESPONSE
+    ```
 
-1. Define your new Integration JSON and save it to a file locally.
+1. Save the base64 encoded `JWT` using `jq` to find `.access_token` (See [./jq](https://jqlang.github.io/jq/) for more info)
 
-```json
-{
-    "display_name": "TrafficLight101",
-    "custom_claims": {
-      "serial_number": "TL1000000101",
-      "has_cyclist_light": "true"
-    }
-}
-```
+    ```bash
+    TOKEN=$(echo -n $RESPONSE | jq -r .access_token)
+    echo $TOKEN
+    ```
 
-2. Generate a token using your pre-existing `Custom Integration` details.
+1. Create a Bearer Token file with `0600` permissions for reference by `curl` commands
 
-```bash
-curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
-    --data-urlencode "grant_type=client_credentials" \
-    --data-urlencode "client_id=${CLIENT_ID}" \
-    --data-urlencode "client_secret=${SECRET}"
-```
+    ```bash
+    echo Authorization: Bearer $TOKEN > rkvst-bearer.txt
+    cat rkvst-bearer.txt
+    ```
 
-The token is found in the `.access_token` field as a base64 encoded [JSON Web Token](https://jwt.io/introduction/).
+## Testing Token Creation
 
-A common method to extract the token is to use `jq`, where `$RESPONSE` is the output of your curl command:
+You can test access to the RKVST API using any of the standard API calls. GETing `Assets` is a simple test.
 
-```bash
-TOKEN=$(echo -n $RESPONSE | jq -r .access_token)
-```
-
-You should then save the token to a local `bearer_token` file with `0600` permissions in the following format:
+If successful, you will see a list of the assets the Integration has access to in the tenancy.
+{{< note >}}
+Note this may be an empty response if no assets are being shared with the user; this is an expected secure by default behavior.
+{{< /note >}}
 
-```bash
-Authorization: Bearer $TOKEN
-```
+Otherwise, check the [Assets OpenAPI Reference](../../api-reference/assets-api/#assets-openapi-reference) for more detailed information on the response codes you may expect if authentication fails and what they mean.
 
-Where `$TOKEN` is the extracted token value.
+### View Existing Assets
 
-3. Submit your new Application JSON to the App Registration API endpoint. 
+To test the creation of the Custom integration and the configuration of the bearer token file (`rkvst-bearer.txt`), query the assets API
 
 ```bash
-curl -X POST \
-     -H "@$BEARER_TOKEN_FILE" \
-     -H "Content-Type: application/json" \
-     -d "@/path/to/jsonfile" \
-     https://app.rkvst.io/archivist/iam/v1/applications
+curl -H "@rkvst-bearer.txt" \
```

Comment:
might be an idea to explicitly add the GET as to make it clear and it aligns with the current API documentation

@SteveLasker

Reply:
Good spot, people who are new to curl or REST might need the examples to be free of assumptions.

Reply:
Thanks @mmwilder26, it was the warning from the `curl` command that GET is the default that caused me to remove it. 
I do like the explicit command.

Reply:
I just realized, the text: 
> Unnecessary use of -X or --request, GET is already inferred.

Is only present when `-v` is also passed.
```bash
curl -v -X GET -H "@rkvst-bearer.txt"     https://app.rkvst.io/archivist/v2/assets | jq
Note: Unnecessary use of -X or --request, GET is already inferred.
```

I've updated to include `-X GET` and remove `-v` as it was overly noisy for the getting started experience.